### ISSUE TITLE
PIP Compile Error

### ISSUE THREAD

**Issue by shandyags** [mail-in-a-box/mailinabox#1080]
ImportError: No module named 'packaging'

----------------------------------------
Cleaning up...
Command /usr/bin/python3 -c "import setuptools, tokenize;__file__='/tmp/pip_build_root/cffi/setup.py';exec(compile(getattr(tokeniz                                    e, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record /tmp/pip-8ruk4aug-record/install-rec                                    ord.txt --single-version-externally-managed --compile failed with error code 1 in /tmp/pip_build_root/cffi
Storing debug log for failure in /root/.pip/pip.log

**Comment by cromulus** [comment]
I have replicated this. 
Will try to put together a PR as this is currently a blocker for me adding new hosts.

**Comment by axd** [comment]
Been there, that's a problem with `python-request` you have do the following:

```
$ sudo apt-get purge python3-requests
$ sudo apt-get install python3-requests python3-pip
```
Later install through `pip` the `packaging` package

`pip3 install packaging`

And see if you get a successful install if so, run again the setup file of MiAB.

Cheers.

**Comment by kenthinson** [comment]
@axd thanks for that! I also had to run 
`pip3 install appears`
In addition to the commands you said.

**Comment by Lucif3r** [comment]
@axd Tried your method. Still getting this error.

**Comment by kenthinson** [comment]
@Lucif3r make sure you use a clan install of ubuntu. I tried it and it would not work until i did a fresh install.

**Comment by JoshData** [comment]
Fixed in v0.21c posted this morning.
